src/frontend/src/views/Data/components/TreeTableNameCell/index.tsx

```diff
@@ -18,12 +31,55 @@ interface Props {
   item: PhyloRun;
 }
 
+const getDateRangeString = (item: PhyloRun): string => {
+  // NOTE: The start date default is covid-specific. We will need to update for
+  // other pathogens
+  const startDate = item.templateArgs?.filterStartDate || "2019-12-23";
+  const endDate =
+    item.templateArgs?.filterEndDate || item.startedDate.slice(0, 10);
```

Comment:
Nit: could you add a brief comment explainer about `item.startedDate` making sense when the filterEndDate is missing? I think it makes sense in the context of actively thinking about this like we are, but in a couple months when I forget that context, I think having the `startedDate` for the `endDate` is just going to seem like a typo rather than a purposeful choice because of it being an implicit end to when the tree had samples available for use.